### DISCUSSION TITLE
Are the schematics for the DevKit open-source?

### DISCUSSION THREAD

**Discussion by MikeOpenHWGroup** [openhwgroup/core-v-mcu-devkit#30] [Q&A: DevKit hardware] [answered]
If the schematics for the CORE-V-MCU DevKit are open-source, where can I find them and what schematic editor is required to view/edit them?

**Comment by n9wxu** [comment, marked as answer]
Absolutely.  The entire devkit design package is available in this repo.  They are in the kicad format.  kicad can be obtained here: https://www.kicad.org/
